content/downloads/releases/111.md

```diff
@@ -10,8 +10,6 @@ Update Site (to be installed into your IDE):\
 Target Update Site (DO NOT INSTALL THIS INTO YOUR IDE. THIS IS MEANT ONLY FOR YOUR TARGET PLATFORM!):\
 **[http://download.eclipse.org/ecp/releases/releases_target_111/](http://download.eclipse.org/ecp/releases/releases_target_111/)**\
 \
-**[Download Target Update Site as a zip file](http://www.eclipse.org/downloads/download.php?file=/ecp/releases/releases_target_111/1110/1110.zip)**\
```

Comment:
This link is working for me. The broken one is on line 8

Reply:
Thank you for double checking.